odpf/optimus/runtime_service.proto

```diff
@@ -773,11 +778,13 @@ message BackupDryRunRequest {
     string resource_name = 3;
     string namespace = 4;
     string description = 5;
-    bool ignore_downstream = 6;
+    bool ignore_downstream = 6 [deprecated=true];
+    repeated string allowed_downstream = 7;
```

Comment:
what if we rename it to `downstream_namespace_filter`? Also add some comments about how this work?

Reply:
sure i can add the comment. for the name, does the concern of naming it `allowed_downstream` is because it is not clear enough that it is representing namespaces? if yes, does `allowed_downstream_namespace` sounds better than `downstream_namespace_filter`? to avoid questions on what is this filter, why it is needed.

Reply:
as discussed, will go with `allowed_downstream_namespaces`

Reply:
hi @kushsharma , have changed to `allowed_downstream_namespaces` and added the comments. please take a look.